### PR TITLE
Storybookjs Site Generation

### PR DESCRIPTION
**Pull request by Alex-Aralis**
This PR does not hook storybook into github pages in any way, or automate site generation. To build the docs statically run `npm run build-storybook`. To live update and server the docs along with starting the ember dev server run `ember run storybook`.

image

### CODE REVIEW

**Review by Alex-Aralis — COMMENTED** [review]
Looks good to me 👍

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    script.src = location.protocol + '//' + host + ':4200' + src;
+    document.getElementsByTagName('head')[0].appendChild(script);
+  })();
+</script>
+<script src="./testem.js"></script>
+<script src="./assets/vendor.js"></script>
+<script>
+  runningTests = true;
+  Ember.testing = true;
+</script>
```
Comment (kevinhinterlong) [inline .storybook/preview-head.html:28]: Is this needed?

File: .storybook/preview-head.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<!-- This file is auto-generated by ember-cli-storybook -->
+<meta
+  name="dummy/config/environment"
+  content="%7B%22modulePrefix%22%3A%22dummy%22%2C%22environment%22%3A%22test%22%2C%22rootURL%22%3A%22%2F%22%2C%22locationType%22%3A%22none%22%2C%22EmberENV%22%3A%7B%22FEATURES%22%3A%7B%7D%2C%22EXTEND_PROTOTYPES%22%3A%7B%22Date%22%3Afalse%7D%2C%22_APPLICATION_TEMPLATE_WRAPPER%22%3Afalse%2C%22_DEFAULT_ASYNC_OBSERVERS%22%3Atrue%2C%22_JQUERY_INTEGRATION%22%3Afalse%2C%22_TEMPLATE_ONLY_GLIMMER_COMPONENTS%22%3Atrue%7D%2C%22APP%22%3A%7B%22LOG_ACTIVE_GENERATION%22%3Afalse%2C%22LOG_VIEW_LOOKUPS%22%3Afalse%2C%22rootElement%22%3A%22%23ember-testing%22%2C%22autoboot%22%3Afalse%7D%2C%22exportApplicationGlobal%22%3Atrue%7D"
```
Comment (kevinhinterlong) [inline .storybook/preview-head.html:4]: Do we need the test environment?

**Review by kevinhinterlong — APPROVED** [review]
Looks good other than I'm not sure we need to be using the test environment

File: .storybook/preview-head.html (hunk 1 above)
Comment (jkusa) [reply .storybook/preview-head.html:28]: Auto-generated

File: .storybook/preview-head.html (hunk 2 above)
Comment (jkusa) [reply .storybook/preview-head.html:4]: same here

**Pull request merged**